src/jit/emitarm64.cpp

```diff
@@ -9285,6 +9480,33 @@ size_t emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE** dp)
             dst += emitOutput_Instr(dst, code);
             break;
 
+        case IF_LS_3F: // LS_3F   .Q.........mmmmm xx.xssnnnnnttttt      Vt Rn Rm
+            elemsize = optGetElemsize(id->idInsOpt());
+            code     = emitInsCode(ins, fmt);
+
+            code |= insEncodeReg_Vt(id->idReg1());  // ttttt
+            code |= insEncodeReg_Rn(id->idReg2());  // nnnnn
+            code |= insEncodeVLSElemsize(elemsize); // ss
+            code |= 0x5000;                         // xx.x         - We only support the one register variant right now
+            code |= insEncodeReg_Rm(id->idReg3());  // mmmmm
```

Comment:
You will want to extend the enum `insOpts` like this:

```
    INS_OPTS_8B  = 16,
    INS_OPTS_16B,
    INS_OPTS_4H,
    INS_OPTS_8H,
    INS_OPTS_2S,
    INS_OPTS_4S,
    INS_OPTS_1D,
    INS_OPTS_2D,

    INS_OPTS_MUL_V1  = 32,   //  multiple structures - One Vector register
    INS_OPTS_MUL_V2,            //  multiple structures - Two consecutive Vector registers
    INS_OPTS_MUL_V3,            //  multiple structures - Three consecutive Vector register variant
    INS_OPTS_MUL_V4,            //  multiple structures - Four consecutive Vector register variant

    INS_OPTS_MSL,     // Vector Immediate (shifting ones variant)
```

Reply:
I was going to log a bug and handle this in a future PR, as we have no intrinsics that will expose these today.

I can certainly add the opts and assert that it is only V1 for now, if that is preferred?

Reply:
However you want, I also believe that Egor will be working in this area.

Reply:
Actually, it looks like we might not need these as the instructions that load to multiple vector registers have different names (e.g. `ld1`, `ld2`, `ld3`, `ld4`).

Reply:
If you're talking about the `enum`, it would still be useful, instructions like `tbl` have the same mnemonic but varying numbers of registers allowed in the list.

Reply:
Good to know. I think we should definitely hold off on adding it until we add an instruction that is capable of using it, however.

Reply:
> Actually, it looks like we might not need these as the instructions that load to multiple vector registers have different names (e.g. ld1, ld2, ld3, ld4).

I think we will need them to support ld1 with multiple vector registers - there are four variant of this instruction.